sytest/pipeline.yml

```diff
@@ -200,22 +200,130 @@ steps:
           image: "matrixdotorg/sytest-dendrite"
           propagate-environment: true
           always-pull: true
-          workdir: "/sytest"
+          workdir: "/src"
           entrypoint: '/bin/sh'
           shell: ["-x", "-c"]
           init: false
+          debug: true
```

Comment:
is this required?

Reply:
I have no idea. I just copied and pasted the pipelines from Dendrite, and `git blame` suggests Amber added it 15 months ago.

Reply:
ok, it's not required. please don't cargo-cult it.